src/service/action-impl.js

```diff
@@ -121,6 +126,39 @@ export class ActionService {
   }
 
   /**
+   * Installs action handler for the specified element.
+   * @param {!Element} target
+   * @param {function(!ActionInvocation)} handler
+   */
+  installActionHandler(target, handler) {
+    const debugid = target.tagName + '#' + target.id;
+    assert(this.isAmpElement_(target), 'AMP element is expected: %s', debugid);
```

Comment:
@dvoytenko correct, tracked in https://github.com/ampproject/amphtml/pull/1031 will continue work on it soon.